optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/scope/PartitionChangeMove.java

```diff
@@ -85,13 +86,13 @@ protected void doMoveOnGenuineVariables(ScoreDirector<Solution_> scoreDirector)
     }
 
     @Override
-    public boolean isMoveDoable(ScoreDirector<Solution_>  scoreDirector) {
+    public boolean isMoveDoable(ScoreDirector<Solution_> scoreDirector) {
         return true;
     }
 
     @Override
-    public PartitionChangeMove<Solution_> createUndoMove(ScoreDirector<Solution_> scoreDirector) {
-        throw new UnsupportedOperationException();
+    public NoChangeMove<Solution_> createUndoMove(ScoreDirector<Solution_> scoreDirector) {
+        return new NoChangeMove<>();
```

Comment:
Because of "PLANNER-611 Move.doMove() must return undo move" this method is now called in `AbstractMove.doMove()` so it must not throw the exception. Returning NoChangeMove seems to me like a good replacement.